airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```
@@ -164,7 +165,7 @@
                     }
                   },
                   {
-                    "title": "zstandard",
+                    "title": "Zstandard",
```

```suggestion
                    "title": "zstandard",
```
These types are always in lower case, I don't think it is necessary to put them in upper case.